src/server/wtr-config.js

```diff
@@ -102,52 +152,24 @@ export class WTRConfig {
 		...passthroughConfig
 	} = {}) {
 
-		if (!this.#cliArgs.group || this.#cliArgs.group === 'default') {
-			if (this.#cliArgs.playwright) {
+		const { watch, manual, group, files, playwright	} = this.#cliArgs;
+
+		if (!group || group === 'default') {
+			if (playwright) {
```

Comment:
Haha, just wrapping their CLI is looking better and better each time I look at these warnings...